src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1669,8 +1669,9 @@ ucs_status_t uct_dc_mlx5_ep_check_fc(uct_dc_mlx5_iface_t *iface,
         fc_entry = &kh_value(&iface->tx.fc_hash, it);
     }
 
-    fc_entry->seq       = iface->tx.fc_seq++;
     fc_entry->send_time = now;
+    fc_entry->seq       = iface->tx.fc_seq++;
+    ucs_assert(iface->tx.fc_seq != UINT64_MAX);
```

Comment:
what corner case does it check?

Reply:
we pass `UINT64_MAX` to `uct_dc_mlx5_fc_remove_ep` function which indicates that no need to check `seq` when getting element from FC hash

Reply:
so you checking that counter on iface does not reach `UINT64_MAX`?

Reply:
> so you checking that counter on iface does not reach UINT64_MAX?

yes, that's right to make sure that check `uct_dc_mlx5_fc_remove_ep` works correctly.
added the comment.